src/utils.ts

```diff
@@ -213,3 +215,15 @@ export function detectBufferEncoding(buffer: Buffer, length = buffer.length): En
 export function denormalizeWinPath(path: string): string {
     return path.replace(/\\/g, "/");
 }
+
+export async function writeFileAsync(fileName: string, data: string): Promise<void> {
+    await new Promise<void>((resolve, reject) => {
+        fs.writeFile(fileName, data, (error?: Error) => {
+            if (error !== null) {
```

Comment:
the mixed types here don't make sense. The optional paramter indicates that `error` may be undefined, but you compare it to `null`.
I guess consistently using `null` here is the right thing to do.